tests/planning_poker/test_consumers.py

```diff
@@ -1,22 +1,54 @@
 from unittest.mock import Mock, patch
 
 import pytest
+from channels.db import database_sync_to_async
+from channels.testing import WebsocketCommunicator
+
+from example.asgi import application
+from planning_poker.models import PokerSession, Story
 
 
 class TestPokerConsumer:
-    def test_poker_session(self, poker_consumer):
-        assert poker_consumer.poker_session.id == 1
-
-    # @pytest.mark.django_db
-    # @pytest.mark.asyncio
-    # async def test_connect(self):
-    #     application = URLRouter([
-    #         url(r'/(?P<poker_session>\w+)/$', PokerConsumer)
-    #     ])
-    #     communicator = WebsocketCommunicator(application, 'ws://test/planning_poker/1/')
-    #     connected, subprotocol = await communicator.connect()
-    #     assert connected
-    #     await communicator.disconnect()
+    def test_poker_session(self, poker_consumer, poker_session):
+        assert poker_consumer.poker_session.id == poker_session.id
+
+    @pytest.mark.asyncio
+    @pytest.mark.django_db
+    @pytest.mark.parametrize('has_active_story', [True, False])
+    async def test_connect(self, has_active_story):
+        poker_session = PokerSession(poker_date='2000-01-01', name='Test Sprint')
+        if has_active_story:
+            story = Story(ticket_number='TEST-1', title='Test Title', description='Test description.')
+            await database_sync_to_async(story.save)()
+            poker_session.active_story = story
+        await database_sync_to_async(poker_session.save)()
+        communicator = WebsocketCommunicator(application, 'ws://test/planning_poker/{}/'.format(poker_session.id))
+        connected, subprotocol = await communicator.connect()
+        assert connected
+        if has_active_story:
+            response = await communicator.receive_json_from()
+            expected_response = {
+                'type': 'send_json',
+                'event': 'story_changed',
+                'data': {
+                    'id': story.id,
+                    'story_label': str(story),
+                    'description': story.description,
+                    'votes': {}
+                }
+            }
+            assert response == expected_response
+        response = await communicator.receive_json_from()
+        expected_response = {
+            'type': 'send_json',
+            'event': 'participants_changed',
+            'data': {
+                'participants': [],
+            },
+        }
+        assert response == expected_response
+        assert communicator.receive_nothing(1)
```

Comment:
```
  /development/planning-poker/tests/planning_poker/test_consumers.py:46: RuntimeWarning: coroutine 'ApplicationCommunicator.receive_nothing' was never awaited
    assert communicator.receive_nothing(1)
```

You obviously have to wait for the result, so the line should be changed to `assert await ...`